app/grandchallenge/cases/image_builders/dicom_4dct.py

```diff
@@ -0,0 +1,191 @@
+import tempfile
+from pathlib import Path
+
+import SimpleITK
+import numpy as np
+import pydicom
+
+from grandchallenge.cases.image_builders import ImageBuilderResult
+from grandchallenge.cases.image_builders.utils import convert_itk_to_internal
+
+NUMPY_IMAGE_TYPES = {
+    "character": SimpleITK.sitkUInt8,
+    "uint8": SimpleITK.sitkUInt8,
+    "uint16": SimpleITK.sitkUInt16,
+    "uint32": SimpleITK.sitkUInt32,
+    "uint64": SimpleITK.sitkUInt64,
+    "int8": SimpleITK.sitkInt8,
+    "int16": SimpleITK.sitkInt16,
+    "int32": SimpleITK.sitkInt32,
+    "int64": SimpleITK.sitkInt64,
+    "float32": SimpleITK.sitkFloat32,
+    "float64": SimpleITK.sitkFloat64,
+}
+
+
+def pixel_data_reached(tag, vr, length):
+    if pydicom.datadict.keyword_for_tag(tag) == "PixelData":
+        return True
+    return False
```

Comment:
Nitpick: this can be reduced to `return pydicom.datadict.keyword_for_tag(tag) == "PixelData"`